src/config/dynamic-theme-fixes.config

```diff
@@ -1232,6 +1232,13 @@ text
 
 ================================
 
+computerhope.com
+
+INVERT
+#text-tool > p > img
```

Comment:
Couldn't find this element.

Reply:
https://www.computerhope.com/cgi-bin/text-tool.pl